compass/landice/tests/dome/run_model.py

```diff
@@ -91,6 +91,9 @@ def __init__(self, test_case, velo_solver, mesh_type, name='run_model',
                             target='../setup_mesh/landice_grid.nc')
         self.add_input_file(filename='graph.info',
                             target='../setup_mesh/graph.info')
+        if velo_solver == 'FO':
+            self.add_input_file(filename='albany_input.yaml',
```

Comment:
@xylar , when adding Albany's configuration file in this way, it shows up in the workdir run directory as a symlink.  My understanding from looking at the code is that using the `filename` argument should add it as an actual file.  Am I reading that wrong, or is this function not behaving properly?

It's not a big deal if this file is a symlink or file, but I think a standard file is preferable in this case.

Reply:
It's functioning properly, `add_input_file` always makes a symlink unless you are downloading a file directly to the work directory from a URL.  There is currently no way to make a copy of an input file from the package but that would be an easy functionality to add.  My preference would be to add a new `copy=False` keyword argument to `add_input_file()` so you could pass `copy=True` if you want to copy rather than symlink.  I'll do this now, and you can review.

Reply:
See https://github.com/MPAS-Dev/compass/pull/138